bot/cogs/custom_ui/page_turn_embed.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+
+import discord
+from discord.ext import commands
+
+
+class PageTurnView(discord.ui.View):
+    """discord.ui.View subclass for page-turn functionality between embeds
+
+    :param ctx: Discord's command context
+    :type ctx: discord.ext.commands.Context
+
+    :param embed_list: List of embeds to page turn
+    :type embed_list: List[discord.Embed]
+
+    :param timeout: (Optional) Float indicating seconds before timeout. (Default=60.0)
+    :type timeout: float
+    """
+
+    def __init__(
+        self, ctx: commands.Context, embed_list: List[discord.Embed], timeout=60.0
+    ):
+        self.ctx = ctx
+        self.embed_list = embed_list
+        self.current_page = 0
+
+        super().__init__(timeout=timeout)
+
+    # Checks that the user who invoked the command is the one interacting
```

Comment:
Suggestion
You can remove the comment or reference the discord docs since we are overloading a function
https://discordpy.readthedocs.io/en/latest/interactions/api.html?highlight=view#discord.ui.View.interaction_check